stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/header/MessageListHeader.kt

```diff
@@ -184,7 +184,7 @@ public fun DefaultMessageListHeaderCenterContent(
     onHeaderTitleClick: (Channel) -> Unit = {},
 ) {
     val title = when (messageMode) {
-        MessageMode.Normal -> ChatTheme.channelNameFormatter.formatChannelName(channel, currentUser)
+        MessageMode.Normal -> ChatTheme.channelNameFormatter.formatChannelName(channel = channel, currentUser = null)
```

Comment:
It is a breaking change.
Customer implement `ChatThem.channelNameFormatter.formatChannelName()` method, and they would be using the current user value. If we pass `null` by default, we are breaking this logic.